sdk/testing/server/fixture.go

```diff
@@ -91,8 +92,9 @@ func NewFixture(t *testing.T, cfgs ...Config) Fixture {
 	// Launch kcp servers and ensure they are ready before starting the test
 	start := time.Now()
 	t.Log("Starting kcp servers...")
-	wg := sync.WaitGroup{}
-	wg.Add(len(servers))
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+	g, ctx := errgroup.WithContext(ctx)
```

Comment:
For the reviewer: error groups fail when the first go routine fails. Before the `wg.Wait()` was blocking forever.